lib/compiler/src/beam_ssa_pre_codegen.erl

```diff
@@ -1526,7 +1526,7 @@ rce_reroute_terminator(#b_switch{list=List0}=Last, Exit, New) ->
 %%  in the exit block following the receive.
 
 recv_fix_common([Msg0|T], Exit, Rm, Blocks0, Count0) ->
-    {Msg,Count1} = new_var('@recv', Count0),
+    {Msg,Count1} = new_var('@ssa_recv', Count0),
```

Comment:
Do we still need this tag (same for `@retval`, `@ssa_stack` et al below)?

Reply:
I have now eliminated the tuple form of variable names, in order to further simplify the naming of variables. The only allowed allowed variable name is now a single atom or a single integer.